dbt/include/databricks/macros/materializations/seeds/seeds.sql

```diff
@@ -40,8 +93,6 @@
     {{ get_csv_sql(create_table_sql, sql) }};
   {% endcall %}
 
-  {% set target_relation = this.incorporate(type='table') %}
```

Comment:
There's more shared code that I can pull out, but I want to leave the existing materialization largely intact, since the whole reason to use the flag is to lower risk of significant changes.  Also, eventually V1 will be deleted.  I pulled out target_relation so that I could do the return statement at the top level.